code_challenges.rb

```diff
@@ -2,48 +2,112 @@
 
 # --------------------1) Create a method that takes in an array and returns an array with only odd numbers sorted from least to greatest.
 
-
-
-
-
 fullArr1 = [4, 9, 0, '7', 8, true, 'hey', 7, 199, -9, false, 'hola']
 # Expected output: [-9, 7, 9, 199]
 fullArr2 = ['hello', 7, 23, -823, false, 78, nil, '67', 6, 'Number']
 # Expected output: [-823, 7, 23]
 
+puts ""
+
+def oddSort arr
+    nums = []
+    arr.filter do |element|
+        if element.is_a? Integer
+            nums << element
+        end
+    end
+    arr = []
+    nums.filter do |num|
+        if num%2 === 1
+            arr << num
+        end
+    end
+    print arr.sort
+    puts ""
+end
```

Comment:
good work! you can also use `p` instead of `print` here so you can eliminate `puts ""` :)